src/core/renderables/types.d.ts

```diff
@@ -189,6 +189,7 @@ export type RenderableCustomField = {
   heading: string;
   apiName: string;
   description: RenderableContent[];
+  pickListValues: string[];
```

Comment:
As I mentioned on the template file, I'm thinking this should be its own section, rather than the direct list of strings.

This type can be updated to be something like

```
pickListValues?: RenderableSection<string[]>
```

All `RenderableSection`s need to have a heading (in this case we can match what Salesforce does and have it be  something like "Possible Values", or something along those lines).